html-block-and-inline/html-block-and-inline/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html><!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <div>
+      <p>Price: <s>$19.99</s> $14.99</p>
+    </div>
+    <div>
+      <p>
+        Part of our new Americana collection, this <i>beautiful</i> blue is
```

Comment:
a better semantic tag for would be `<em>` instead of `<i>` - I talked about the reasoning in the recording